### PR TITLE
Add limit active sessions adaptive auth doc

### PR DESCRIPTION
**Pull request by sherenewso2**

### CODE REVIEW

File: en/docs/samples/limiting-active-sessions-adaptive-auth.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Configure Limiting Active User Sessions 
+
+This page guides you through setting up active user session limiting based on particular criteria for a sample application using sample authenticators.
```
Comment (gomathyk) [inline en/docs/samples/limiting-active-sessions-adaptive-auth.md]: Shall we change this ?
This page guides you through setting up an active user session for a sample application which is limited based on a particular criteria, using sample authenticators.

File: en/docs/samples/limiting-active-sessions-adaptive-auth.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+4.  Expand **Script Based Conditional Authentication**.
+
+5.  Click **Templates** on the right side of the **Script Based Conditional Authentication** field and then click **Concurrent-Session-Management**. 
+
+    ![limit-active-sessions-template](../../assets/img/samples/limit-active-sessions-template.png)
+
+7.  Click **Ok**. The authentication script and authentication steps
+    are configured. 
+    
+    The authentication script defines a conditional step that executes session handling prompt only if the user belongs to an 'admin' or 'manager' role.  Here you can specify the value of `MaxSessionCount` variable to indicate the maximum number of allowed sessions. The default value is 1. For the purpose of this demo, change the value to 3.
```
Comment (gomathyk) [inline en/docs/samples/limiting-active-sessions-adaptive-auth.md]: maximum number of sessions allowed.

File: en/docs/samples/limiting-active-sessions-adaptive-auth.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+7.  Click **Ok**. The authentication script and authentication steps
+    are configured. 
+    
+    The authentication script defines a conditional step that executes session handling prompt only if the user belongs to an 'admin' or 'manager' role.  Here you can specify the value of `MaxSessionCount` variable to indicate the maximum number of allowed sessions. The default value is 1. For the purpose of this demo, change the value to 3. 
+
+    ![limit-active-sessions](../../assets/img/samples/limit-active-sessions.png)
+    
+    !!! note
+        You can configure the `MaxSessionCount` variable via the deployment.toml file in the  `<IS_HOME>/repository/conf/` directory as well. Priority will be given to the configuration in the adaptive authentication script. To configure the `MaxSessionCount` variable through `deployment.toml` file, append the following configuration with the intended value for `MaxSessionCount`.
```
Comment (gomathyk) [inline en/docs/samples/limiting-active-sessions-adaptive-auth.md]: `deployment.toml`

variable through **the** `deployment.toml`

File: en/docs/samples/limiting-active-sessions-adaptive-auth.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+2.  Click **Login** and enter admin/admin credentials.
+    ![pickup-sign-in.png](../../assets/img/samples/pickup-sign-in.png)
+
+3.  Repeat the previous two steps in three different web browsers, e.g. Firefox, Safari, and Opera.
+    ![view-active-sessions-adaptive-auth](../../assets/img/samples/view-active-sessions-adaptive-auth.png)
+
+4.  Now you can either terminate one or more active sessions or deny the login. 
+
+    !!! tip 
+        -   If you select and terminate the active sessions exceeding the maximum limit, you will be naviagated to the application home page. Otherwise you will be re-prompted until the minimum required number of sessions are terminated.
```
Comment (gomathyk) [inline en/docs/samples/limiting-active-sessions-adaptive-auth.md]: navigated

File: en/docs/samples/limiting-active-sessions-adaptive-auth.md (hunk 1 above)
Comment (sherenewso2) [reply en/docs/samples/limiting-active-sessions-adaptive-auth.md]: That would give a different idea as we are not setting up an active user session for a sample app. 

Will change it to this to make it better:

This page guides you through setting up active user session limiting for a sample application based on a particular criteria, using sample authenticators.

**Pull request merged**